src/utils/constants/environment.ts

```diff
@@ -0,0 +1,4 @@
+export const TENANT_ID = process.env.TENANTID
+  ? process.env.TENANTID
+  : 'ten_I9TW3ncG';
+export const TENANT = process.env.TENANT ? process.env.TENANT : 'planet';
```

Comment:
I always thought `ten_I9TW3ncG` is a tenant named `ttc` while the tenant named `planet` has the id `ten_NxJq55pm`? Now which tenant should be the default one?

Reply:
This is what our admin interface says:

<img width="325" alt="Bildschirmfoto 2021-12-14 um 12 32 48" src="https://user-images.githubusercontent.com/1532418/145990680-c8f6fa47-4360-4bf1-a85a-7e7de3373453.png">

And this is how these tenants named are used in our source:
<img width="413" alt="Bildschirmfoto 2021-12-14 um 12 34 20" src="https://user-images.githubusercontent.com/1532418/145990937-c1ca8d34-c174-45c1-b2ce-891cecf1ae1f.png"><img width="417" alt="Bildschirmfoto 2021-12-14 um 12 34 38" src="https://user-images.githubusercontent.com/1532418/145990925-2339b59f-c446-424c-b1ea-3d30b52dc979.png">

Reply:
Let's stick to planet as default according to `tenant.config.js`. I have been using `ten_I9TW3ncG` for planet tenant since forever. hence the confusing.